### DISCUSSION TITLE
can the "standard_charge | negotiated_percentage" field contain anything other than a percentage of billed charges?

### DISCUSSION THREAD

**Discussion by nmaylevc** [CMSgov/hospital-price-transparency#133] [Q&A] [answered]
Can the "standard_charge | negotiated_percentage" field contain anything other than a percentage of billed charges?

Can that field be used for the percentage of a fee schedule or the percentage of Medicare?

**Comment by nmaylevc** [comment]
I found this language in the [FAQ](https://www.cms.gov/files/document/hospital-price-transparency-frequently-asked-questions.pdf) and it is pretty clearly saying that if the rate is a percentage of Medicare, then the dollar value should be calculated and put in the MRF:

> **In cases where the hospital has negotiated a payer-specific negotiated charge based on the Medicare or Medicaid FFS rate, can the hospital simply indicate that the price of the hospital item/service is set to the Medicare or Medicaid rate instead of reporting a specific dollar value?**
> No. The payer-specific negotiated charge is defined for purposes of the Hospital Price Transparency Final Rule as the charge that a hospital has negotiated with a third-party payer for an item or service, including a service package, and the hospital should list that standard charge. For example, if your hospital has negotiated a payer-specific negotiated charge for a service package that equals 200% of the Medicare FFS reimbursement rate for MS-DRG 123, then your hospital should determine the Medicare reimbursement rate for DRG 123, multiply it by 2 and indicate the resulting amount as its payer-specific negotiated charge for that service
package.

The description in the data dictionary for this field should clarify what the number is a percentage of. This is what the data dictionary currently says:
> Payer-specific negotiated charge (expressed as a percentage) that a hospital has negotiated with a third-party payer for an item or service.

**Comment by RidenourK** [comment]
@nmaylevc. Thank you for your question. 

In the CY 2024 OPPS/ASC Final Rule, we indicated that hospitals are required to display the standard charges as they are established, such that, if the hospital established a standard charge as a dollar amount, the hospital would display the standard charge as a dollar amount. If the hospital has established a standard charge as a percentage or algorithm such that a standard dollar amount is not available, then the hospital would display the standard charge as a percentage or algorithm. We indicated we anticipate that most if not all payer-specific negotiated charges will fall into one of three categories, depending on how a hospital has established them: (1) standard dollar amount, (2) standard algorithm or percentage, or (3) hybrid where a standard dollar amount can be identified but the final allowed amount is dependent on additional variables. An example of where we would expect to see a standard charge in dollars would be standard charges established under a fee schedule or where an identifiable dollar amount has been established for an item or service (for example, a per diem rate, a gross charge for an itemized item or service, or a cash discounted price for a service package). An example of a where we would expect to see a standard charge expressed as an algorithm would be when a hospital has negotiated a reimbursement for defined service packages (for example, hip replacement or colonoscopy) that are based on differential percentages of total billed (gross) charges (for example, 50 percent of total billed charges for hip replacement and 75 percent of total billed charges for colonoscopy). A hybrid would be a situation in which the hospital has established both a standard charge in dollars and there are additional variables that would modify the negotiated rate for a particular item or service. For example, a hospital may have established a payer-specific negotiated charge under the MS-DRG methodology where an adjusted base rate in dollars has been established for each DRG code, but the adjusted base rate may be further modified due to certain variable factors (such as outlier cases or transfers). If the hospital's payer-specific negotiated charge is based on a percentage or algorithm within which no standard dollar amount can be determined, then the hospital should specify what percentage or algorithm determines the dollar amount for the item or service. Please refer to discussion beginning at [88 FR 82097.](https://www.federalregister.gov/d/2023-24293/p-4970)

**Reply by nmaylevc** [reply]
The language quoted from the discussion in the Federal Register is interesting, but doesn't really answer the question asked: what does the negotiated_percentage refer to?  Can it be anything other than a percentage of billed charges?

The Federal Register also states "If the hospital has established a standard charge as a percentage or algorithm such that a standard dollar amount is not available".  See below.  In what cases would a dollar amount not be available?  If the rate is contracted as a percentage of Medicare, then the dollar amount would be calculable and available.  

If it is a percentage of a fee schedule, then that would also be calculable as a dollar value.  In what cases is it not calculable other than a percent of total billed charges for an item like one of the types of DRG codes?

Allowing hospitals to put in a percentages with no clear definition as to what the charge is a percentage of, would not seem to meet the intent of the rule or the language of the final rule [which states](https://www.federalregister.gov/d/2023-24293/p-5868) (bold added by me): "If the standard charge is based on a percentage or algorithm, the machine-readable file (MRF) **must also describe the percentage** or algorithm that determines the dollar amount for the item or service".  A percentage alone is not a sufficient description.

[Register](https://www.federalregister.gov/d/2023-24293/p-4970)
> We clarify that allowing hospitals to display a payer-specific negotiated charge as a standard algorithm is appropriate to the extent a standard algorithm is the manner in which hospitals establish their standard charges with third party payers. Hospitals are required to display the standard charges as they are established, such that, if the hospital established a standard charge as a dollar amount, the hospital would display the standard charge as a dollar amount. If the hospital has established a standard charge as a percentage or algorithm such that a standard dollar amount is not available, then the hospital would display the standard charge as a percentage or algorithm.

**Reply by nmaylevc** [reply]
In addition, the language on GitHub says (bold added):

> fee schedule: The payer-specific negotiated charge is based on a fee schedule. Examples of common fee schedules include Medicare, Medicaid, commercial payer, and workers compensation. The dollar amount that is based on the indicated fee schedule should be encoded into the Payer-specific Negotiated Charge: Dollar Amount data element. **For standard charges based on a percentage of a known fee schedule, the dollar amount should be calculated and encoded in the Payer-specific Negotiated Charge**: Dollar Amount data element.

That language doesn't preclude also "encoding" the percentage, but if you combine it with the language in the Additional Notes for Percentage (below), it seems to narrow the use of percentage to only scenarios where the dollar amount can't be calculated i.e. percentage of billed charges and not a standard fee schedule.

>Additional Notes for Percentage
>Information for this data element should be encoded only when the payer-specific negotiated charge has been established as a percentage and no standard dollar amount can be calculated. 

We are seeing files that are using the percentage for other values.  See the image below:
<img width="1496" alt="Screenshot 2024-08-28 at 1 10 54 PM" src="https://github.com/user-attachments/assets/103b93c6-b55c-4102-82b4-fe0d396e9de7">

**Comment by RidenourK** [comment, marked as answer]
@nmaylevc. Thank you for your question. 

As explained in the CY2024 OPPS/ASC final rule, hospitals use different methods to establish standard charges for items and services, resulting in charge/item and charge/service combinations that are often unique to that hospital. Therefore, although CMS has recently taken steps toward standardization, each hospital will continue to have some discretion related to how it chooses to encode its standard charge information (including information related to payer-specific negotiated charges) within its machine-readable file, so long as the file conforms to the CMS template layouts and data specifications as described at 45 CFR 180.50(c)(2).  

However, as indicated in the data dictionary, if a "payer specific negotiated charge" is encoded as a dollar amount, percentage, or algorithm then a corresponding valid value for the payer name, plan name, and standard charge methodology must also be encoded and if the "standard charge methodology" encoded value is "other", there must be a corresponding explanation found in the "additional notes" for the associated payer-specific negotiated charge.
